src/EnergyPlus/OutputReportTabular.cc

```diff
@@ -5245,6 +5245,18 @@ namespace OutputReportTabular {
         //--------------------
         //     ANNUAL
         //--------------------
+        // HVAC annual heating by ATU
```

Comment:
So what actually ended up causing the fix here?  Was it moving this block up above the zone loop?  Or aggregating ATUHeat using `+=` instead of just an assignment?  Or the altered calculation of `total` below?

Reply:
The actual fix is that the aggregation for the ZoneEqHt and ZoneEqCl now occurs in the `GatherHeatGainReport()` function at the HVAC time step level, instead of originally calculated by a subtraction of two annual sum in the `FillRemainingPredefinedEntries()` after all the time step accumulation was done (around Line 6764-6766 in `OutputReportTabular.cc`). 
`
-        PreDefTableEntry(pdchSHGSAnHvacHt, "Total Facility", (totalHvacHt - totalHvacATUHt) * convertJtoGJ, 3);
-        PreDefTableEntry(pdchSHGSAnHvacCl, "Total Facility", (totalHvacCl - totalHvacATUCl) * convertJtoGJ, 3);
+       PreDefTableEntry(pdchSHGSAnZoneEqHt, "Total Facility", totalZoneEqHt * convertJtoGJ, 3);
+       PreDefTableEntry(pdchSHGSAnZoneEqCl, "Total Facility", totalZoneEqCl * convertJtoGJ, 3); 
`

The original method would cause some `canceling' heating and cooling not counted toward the total sum. The revised method does this accumulation of heating and cooling at system time step level, so that the two clearly separated and logged in for each system time step, which both heating and cooling accumulate to get their totals. 

The block order change (ATU block moved up) served as facilitating this accumulating this change in the time step level, in that the ATU heat and cool for each zone is known before the ZoneEq can be calculated through a split subtraction. This can be done without moving the blocks, but would then require creating a new temporary storing variables adding an extra block to do the splitting after both the zone block and the ATU block finishes.

Reply:
@Myoldmopar Regarding 

> Or aggregating ATUHeat using += instead of just an assignment?

That change actually fixes a different bug where this report was wrong for any zone with more than one terminal unit. But I never posted a new issue for that or created a test file to prove it.